redwood-layout-view/src/main/kotlin/app/cash/redwood/layout/view/YogaLayout.kt

```diff
@@ -71,6 +67,11 @@ internal class YogaLayout(
   }
 
   private fun calculateLayout(widthSpec: Int, heightSpec: Int) {
+    rootNode.requestedWidth = Size.Undefined
+    rootNode.requestedMaxWidth = Size.Undefined
+    rootNode.requestedHeight = Size.Undefined
+    rootNode.requestedMaxHeight = Size.Undefined
+
```

Comment:
Is there any downside to not specifying the requested sizes?

Reply:
This just clears them out so that we can set the ones we want below. Otherwise it ends up accumulating both exact and max values when it’s repeatedly measured.